server/src/main/java/facultad/trendz/model/Comment.java

```diff
@@ -0,0 +1,76 @@
+package facultad.trendz.model;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Table(name = "comment")
+public class Comment {
+
+    @Id
+    @GeneratedValue(strategy= GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(length=10000)
+    private String content;
+
+    private Date date;
+
+    @ManyToOne(cascade = CascadeType.ALL)
```

Comment:
CascadeType.ALL va a implicar que cuando elimines un comment, eso va a eliminar el post en donde está contenido también. Es ideal en estos casos mapear esta asociación desde la entidad padre, osea poner el cascade y el mapped by en el Post, y no en el Comment
Leete las primeras 3 respuestas de [este post](https://stackoverflow.com/questions/13027214/what-is-the-meaning-of-the-cascadetype-all-for-a-manytoone-jpa-association) de SO y lo vas a entender mejor